dm/dm/proto/dmworker.proto

```diff
@@ -63,109 +70,111 @@ message QueryStatusResponse {
-//         transfers to Running when resuming is requested from external
-//         transfers to Stopped when requested from external
-// Stopped: indicates the processing is stopped, and can not be resume (or re-run) again
+//          transfers to Finished when sub task processing completed (no Syncer
+//          used)
+// Paused: indicates the processing is paused, and can be resume from external
+// request
+//         is transferred from Running when error occurred or requested from
+//         external transfers to Running when resuming is requested from
+//         external transfers to Stopped when requested from external
+// Stopped: indicates the processing is stopped, and can not be resume (or
+// re-run) again
 //          is transferred from Running / Paused when requested from external
 //          can not transfer to any stages
 // Finished: indicates the processing is finished, and no need to re-run
 //           is transferred from Running when processing completed
 //           should not transfer to any stages
 enum Stage {
-    InvalidStage = 0; // placeholder
-    New = 1;
-    Running = 2;
-    Paused = 3;
-    Stopped = 4;
-    Finished = 5;
-
-    Pausing = 6;
-    Resuming = 7;
-    Stopping = 8;
+  InvalidStage = 0; // placeholder
+  New = 1;
+  Running = 2;
+  Paused = 3;
+  Stopped = 4;
+  Finished = 5;
+
+  Pausing = 6;
+  Resuming = 7;
+  Stopping = 8;
 }
 
 // CheckStatus represents status for check unit
 // adds fields later
 message CheckStatus {
-    bool  passed = 1;
-    int32 total = 2;
-    int32 successful = 3;
-    int32 failed = 4;
-    int32 warning = 5;
+  bool passed = 1;
+  int32 total = 2;
+  int32 successful = 3;
+  int32 failed = 4;
+  int32 warning = 5;
 
-    bytes detail = 6;
+  bytes detail = 6;
 }
 
 // DumpStatus represents status for dump unit
 // add fields later
 message DumpStatus {
-    int64 totalTables = 1;
-    double completedTables = 2;
-    double finishedBytes = 3;
-    double finishedRows = 4;
-    double estimateTotalRows = 5;
+  int64 totalTables = 1;
+  double completedTables = 2;
+  double finishedBytes = 3;
+  double finishedRows = 4;
+  double estimateTotalRows = 5;
 }
 
 // LoadStatus represents status for load unit
 message LoadStatus {
-    int64 finishedBytes = 1;
-    int64 totalBytes = 2;
-    string progress = 3;
-    string metaBinlog = 4;
-    string metaBinlogGTID = 5;
+  int64 finishedBytes = 1;
+  int64 totalBytes = 2;
+  string progress = 3;
+  string metaBinlog = 4;
+  string metaBinlogGTID = 5;
 }
 
-// ShardingGroup represents a DDL sharding group, this is used by SyncStatus, and is differ from ShardingGroup in syncer pkg
-// target: target table name
-// DDL: in syncing DDL
-// firstPos: first DDL binlog pos for this group
-// synced: synced source tables
-// unsynced: unsynced source tables
+// ShardingGroup represents a DDL sharding group, this is used by SyncStatus,
+// and is differ from ShardingGroup in syncer pkg target: target table name DDL:
```

Comment:
please keep the format

Reply:
ok i will do this in next commit, do you have other sugguestion about the update cfg logic?